README.md

```diff
@@ -11,7 +11,7 @@
 
 ## Live Demo
 
-[comming soon]() 🙂
+[Go Live](https://tadesse-alemayehu.github.io/Awesome-books/) 🙂
 
 
 ## Getting Started
```

Comment:
- [ ] Please it will be nice to update your `readme file` with explicit steps to clone and run your app locally as this will be of help to non technical users, and some of them are potential recruiters. For example; 
`git clone git@github.com:Tadesse-Alemayehu/Awesome-books.git`
`cd Awesome-books`
`npm install`
`npm start` .....

Reply:
@t-yanick thank you for the review, we will implement the changes right away.